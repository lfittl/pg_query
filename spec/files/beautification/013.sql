SELECT a FROM b join c on b.d = c.e AND b.f = c.g
