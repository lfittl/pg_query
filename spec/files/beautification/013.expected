SELECT
  "a"
FROM
  "b"
  JOIN "c" ON "b"."d" = "c"."e" AND
              "b"."f" = "c"."g"
